app.py

```diff
@@ -249,10 +241,26 @@ def download_manifest(self):
 
         return manifest_io.getvalue().decode('utf-8')
 
+    def audit_log(self, event: str):
+
+        log_data = f'{datetime.now()} | {event} | {request.remote_addr} | {json.dumps(dict(request.headers))}\n'
```

Comment:
I suggest adding the hash of the file here.

Reply:
Implemented in the new PR here https://github.com/tarantool/rocks.tarantool.org/pull/45/files#diff-568470d013cd12e4f388206520da39ab9a4e4c3c6b95846cbc281abc1ba3c959R258